tests/testthat/test-misc.R

```diff
@@ -1,5 +1,8 @@
 context("test-misc.R")
 
+swe_set_ephe_path('')
```

Comment:
I am sorry, but this makes sense only on your machine where the SE files are installed in a directory that SE expects. That is not the case on my machine and more importantly will never be the case on the machines used for automatic testing (CRAN, Travis, Appveyor). I will revert this commit.